README.md

```diff
@@ -1,12 +1,46 @@
-# ibexa/rector
+# Ibexa DXP Rector
 
 This package is part of [Ibexa DXP](https://ibexa.co).
 
-To use this package, [install Ibexa DXP](https://doc.ibexa.co/en/latest/install/).
-
 This package provides a set of [Rector](https://getrector.com/) rules to allow automatic upgrades between
 [Ibexa DXP](https://ibexa.co) versions.
 
+To use this package, [install Ibexa DXP](https://doc.ibexa.co/en/latest/install/) and follow installation instructions
+below.
+
+## Installation
+
+```
+composer require --dev ibexa/rector:~5.0.x-dev
+```
+
+## Usage
+
+1. Create `./rector.php` file in your project, with the following contents, adjusted to your project structure:
```

Comment:
I suggest adding a recipe so that the file is created when the package is installed (this will make life easier for our partners)